cpanfile

```diff
@@ -1,17 +1,10 @@
-requires 'Class::Accessor::Fast';
-requires 'Class::Data::Inheritable';
 requires 'DBI';
-requires 'Danga::Socket';
 requires 'File::Pid';
+requires 'Module::Load';
 requires 'Parallel::ForkManager';
 requires 'Proc::Daemon';
 requires 'TheSchwartz';
 requires 'TheSchwartz::Simple';
-requires 'TheSchwartz::Worker';
```

Comment:
I believe TheSchwartz::Worker is still used at https://github.com/hatena/WorkerManager/blob/25bf2f435a9c325d98748ea64375db9b80379088/lib/WorkerManager/Worker/TheSchwartz.pm#L4 . Can we remove this line?

Reply:
I guess requiring `TheSchwartz` install `TheSchwartz:Worker`, but I prefer explicitly declaring this dependance, but I don't remember the best practice for cpanfile anymore :scream:

Reply:
Yes `TheSchwartz` (distribution) has `TheSchwartz::Worker` (module), we don't need to declare in cpanfile.